src/upload/admin/controller/extension/payment/webpay_rest.php

```diff
@@ -156,46 +155,33 @@ public function index() {
         $healthcheck = json_decode($hc->printFullResume(), true);
-
-        $lh = new LogHandler();
-        $loghandler = json_decode($lh->getResume(), true);
-
+        $logHandler = new LogHandler();
         $data['hc_data'] = $hc->printFullResume();
         $data['healthcheck'] = $healthcheck;
-        $data['lg_data'] = $lh->getResume();
-        $data['loghandler'] = $loghandler;
-
-        if (isset($loghandler['last_log']['log_content'])) {
-            $data['res_logcontent'] = json_encode($loghandler['last_log']['log_content']);
-            $data['log_file'] = $loghandler['last_log']['log_file'];
-            $data['log_file_weight'] = $loghandler['last_log']['log_weight'];
-            $data['log_file_regs'] = $loghandler['last_log']['log_regs_lines'];
+        $data['lg_data'] = json_decode($logHandler->getResume(), true);
+
+
+        if (isset($data['lg_data']['last_log']['log_content'])) {
+            $data['res_logcontent'] = json_encode($data['lg_data']['last_log']['log_content']);
+            $data['log_file'] = $data['lg_data']['last_log']['log_file'];
+            $data['log_file_weight'] = $data['lg_data']['last_log']['log_weight'];
+            $data['log_file_regs'] = $data['lg_data']['last_log']['log_regs_lines'];
         } else {
-            $data['res_logcontent'] = $loghandler['last_log'][0];
+            $data['res_logcontent'] = $data['lg_data']['last_log'][0];
             $data['log_file'] = json_encode($data['res_logcontent']);
             $data['log_file_weight'] = $data['log_file'];
             $data['log_file_regs'] = $data['log_file'];
         }
 
-        if ($loghandler['config']['status'] === false) {
-            $data['estado_logs'] = "<span class='label label-warning'>Desactivado sistema de Registros</span>";
-        } else {
-            $data['estado_logs'] = "<span class='label label-success'>Activado sistema de Registros</span>";
-        }
-
-        $data['log_list'] = $loghandler['logs_list'];
-        $data['log_dir'] = stripslashes(json_encode($loghandler['log_dir']));
-        $data['log_count'] = json_encode($loghandler['logs_count']['log_count']);
-        $data['tb_max_logs_days'] = $loghandler['config']['max_logs_days'];
-
-        $data['tb_max_logs_weight'] = $loghandler['config']['max_log_weight'];
+        $data['log_list'] = $data['lg_data']['logs_list'];
+        $data['log_dir'] = stripslashes(json_encode($data['lg_data']['log_dir']));
+        $data['log_count'] = json_encode($data['lg_data']['logs_count']['log_count']);
 
         $data['url_create_pdf_report'] = '../catalog/controller/extension/payment/libwebpay_rest/CreatePdf.php?document=report';
         $data['url_create_pdf_php_info'] = '../catalog/controller/extension/payment/libwebpay_rest/CreatePdf.php?document=php_info';
-        $data['url_check_conn'] = '../catalog/controller/extension/payment/libwebpay_rest/CheckConn.php';
+        $data['url_check_conn'] = '../system/library/Transbank/utils/CheckConn.php';
```

Comment:
The path is `util`? Maybe `Util` is better

Reply:
Route modification will be performed, adapting it to the use of the newly method called checkConnection
This method will replace the functionality that the CheckConn.php file had.